Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -601,7 +601,7 @@ def genS(fragment,howMuch):
 steps['AMPT_PPb_5020GeV_MinimumBias']=merge([{'-n':10},step1PPbDefaults,genS('AMPT_PPb_5020GeV_MinimumBias_cfi',Kby(9,100))])
 
 ## pPb Run2
-step1PPbDefaultsUp15={'--beamspot':'Run2PPbBoost','--conditions':'auto:run2_mc_pa','--eventcontent':'RAWSIM', '--era':'Run2_2016_pA'}
+step1PPbDefaultsUp15={'--beamspot':'RealisticPPbBoost8TeV2016Collision','--conditions':'auto:run2_mc_pa','--eventcontent':'RAWSIM', '--era':'Run2_2016_pA'}
```

Comment:
@davidlange6 :  Yes, @jrcastle is taking charge of this.  We are currently reconstructing a sample with a temporary BS, based on this vertex smearing, to evaluate the reco BS.  Once that's done we'll update the GT, as well as the autocond's.  We prefer to do this in two steps, since the GEN-SIM can already start once the vertex smearing is integrated.